tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -588,4 +740,43 @@ INSTANTIATE_TEST_SUITE_P(
             Sizes({1, 1}),
             Sizes({b, 1, n}))));
 
+INSTANTIATE_TEST_SUITE_P(
```

Comment:
I still need to think about a good naming scheme: Underscore separated shapes looked too verbose and for filtering we will need the exact shapes.